snips/run.sh

```diff
@@ -36,24 +53,30 @@ if [ "$MQTT_BRIDGE" == "true" ]; then
           echo "password $PASSWORD"
       } >> /etc/mosquitto.conf
     fi
-    
+
     {
-        echo "topic # OUT"
-        echo "topic # IN hermes/ hermes/"
+        echo "topic hermes/intent/# out"
```

Comment:
Why do we need to be so specific? Can't we just bridge everything under `hermes/` ?

Reply:
Snips uses MQTT to actually transmit the audio PCM frames, so it seems inefficient to me to clutter the hassio mqtt broker with constant binary audio frames which are useless to any other app.